data/accidentalProphetStory.xml

```diff
@@ -19,8 +19,8 @@ The devotion he shows gives you renewed faith in your cause. You agree to take h
 					<quantModifier id="charityMetric" absolute="false">+1</quantModifier>
 				</outcome>
 			</choice>
-		</storyNode>
-		
+			<teaserImage>../data/images/wealth.png</teaserImage></storyNode>
```

Comment:
Doesn't matter for now, but I'd probably put the XML for the teaser image right under the teaser text in the future. :)